server/vertx/src/main/scala/sttp/tapir/server/vertx/VertxFutureServerInterpreter.scala

```diff
@@ -44,7 +45,7 @@ trait VertxFutureServerInterpreter extends CommonServerInterpreter {
     * @return A function, that given a router, will attach this endpoint to it
     */
   def routeRecoverErrors[I, E, O](e: Endpoint[I, E, O, Any])(
-    logic: I => Future[O]
+      logic: I => Future[O]
```

Comment:
I think my editor is enforcing the `.scalafmt.conf` in a few places it wasn't already.

Reply:
yeah that's possible, no worries :)